imdb/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url
+
+from  . import  views
+
+urlpatterns = {
+    url(r'^search/', views.search_movie, name="search_movie"),
+    url(r'^get/([a-z0-9A-z]+)/', views.exact_match),
```

Comment:
here this denotes that any combination of lowercase alphabets digits and uppercase alphabets in the given order . when searching the url to check the matched url